MIGRATION.md

```diff
@@ -1,123 +0,0 @@
-# Migrations
```

Comment:
This file should not be removed, it documents how consumers should migrate major versions.